lib/cn/spill.c

```diff
@@ -244,6 +244,9 @@ cn_subspill(
             return 0; /* Nothing to do */
     }
 
+    if (!sctx->more || (key_obj_cmp(&sctx->curr->kobj, &ekobj) > 0 && !sctx->pt_set))
+        return 0; /* Nothing to do */
+
```

Comment:
Does it make sense to write this as   (!sctx->pt_set && key_obj_cmp() > 0)  ???

Reply:
But that replicates the `pt_set` check, which IMO makes it prone to breakage in future changes to this code.  I would just put the simple term first like Greg suggested at the top of his comment.

Reply:
Yeah, I could just split the checks at the OR. Wouldn't even need to check for pt_set in the first early exit..